DockerfileOCIR

```diff
@@ -0,0 +1,13 @@
+FROM maven:3.8.6-openjdk-8 AS build
+WORKDIR /app
+COPY pom.xml .
+COPY src ./src
+RUN mvn clean install -DskipTests
+
+FROM iad.ocir.io/id204we8d65n/openjdk8-alpine:1.0
```

Comment:
in which tenancy we have this image? we should some how indicate that it should not be deleted for some reason.

Reply:
This image is in dev test tenancy.